terminalfighter.py

```diff
@@ -99,11 +109,38 @@ def terminalfighter(screen):
         else:
             print("WARNING: Cannot keep up with 30FPS update time!")
 
-if __name__ == '__main__':
-    GAME_WIDTH = 1000
-    GAME_HEIGHT = 700
+def draw_background():
+    background_surface = pygame.Surface((GAME_WIDTH, GAME_HEIGHT))
+    background_surface.fill(BLACK)
+
+    line_separation = 25
+    line_width = 1
 
-    screen = pygame.display.set_mode(
-        (int(GAME_WIDTH), int(GAME_HEIGHT)))
+    for i in range(line_separation, GAME_WIDTH, line_separation):
+        pygame.draw.line(background_surface,
+                         DARK_GREEN,
+                         (i, 0),
+                         (i, GAME_HEIGHT),
+                         line_width)
 
-    terminalfighter(screen)
+    for i in range(line_separation, GAME_HEIGHT, line_separation):
+        pygame.draw.line(background_surface,
+                         DARK_GREEN,
+                         (0, i),
+                         (GAME_WIDTH, i),
+                         line_width)
+
+    return background_surface
+
+
+if __name__ == '__main__':
+    # Start profiling
+    pr = cProfile.Profile()
+    pr.enable()
```

Comment:
Is this something that should always be enabled? Does it have a very high overhead? Could add an environment variable or CLI argument to enable if it is.

Reply:
https://docs.python.org/3.8/library/profile.html#introduction-to-the-profilers
Apparently it has a "reasonable overhead", but I do agree, I think it might be nice to have a CLI argument to enable it, and have it off by default

something like --profile or something to enable this profile

Reply:
done